File: JavaScript/Contagem.js
// const prompt = require("prompt-sync")();
const N = Number(prompt("Digite o número de notas: "))
let contador = 0;

for (let i = 1; i <= N; i++) {
    let nota = Number(prompt("Digite uma nota: "))

    if (nota >= 50) contador++
}

console.log(`O número de alunos aprovados foi ${contador}`)

File: Java/Contagem/src/Contagem.java
import java.util.Scanner;

public class Contagem {

    public static void main(String[] args) {
        System.out.println("Digite o número de notas: ");
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();

        System.out.println("O total de alunos aprovados foi " + calcularAprovados(N));
    }
    public static int calcularAprovados(int numeroDeNotas) {
        Scanner scanner = new Scanner(System.in);
        int contador = 0;

        for (int i = 1; i <= numeroDeNotas; i++) {
            System.out.println("Digite a próxima nota: ");
            int nota = scanner.nextInt();

            if (nota >= 50) contador++;
        }
        return contador;
    }
}